src/common/github/pull-request-test-workflow.ts

```diff
@@ -72,13 +79,17 @@ export class PullRequestTest extends Component {
 
     workflow.addJobs({
       lint: runScriptJob({command: 'lint', ...commonJobProps}),
-      test: runScriptJob({command: 'test', ...commonJobProps}),
+      'unit-tests': runScriptJob({command: 'test', ...commonJobProps}),
       typecheck: runScriptJob({command: 'typecheck', ...commonJobProps}),
     })
 
-    if (options.lighthouse) {
+    if (options.isLighthouseEnabled) {
```

Comment:
Since this workflow is applied in `cdk` and `apollo-server` templates, please disable `playwright` there. See the [lighthouse disable](https://github.com/ottofeller/templates/blob/main/src/cdk/index.ts#L78) in `cdk` for reference. For `apollo-server` it is not disabled but seems like we do need it disabled as well.